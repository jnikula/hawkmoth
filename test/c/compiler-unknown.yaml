test:
- extension
directives:
- domain: c
  directive: autodoc
  arguments:
  - doc.c
conf-overrides:
  hawkmoth_compiler: false
errors: compiler-unknown.stderr
expected: doc.rst
