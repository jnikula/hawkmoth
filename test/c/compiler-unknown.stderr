WARNING: get_include_args: incompatible c compiler ('false')
WARNING: get_include_args: incompatible c++ compiler ('false')
